functions/DynamicParams.ps1

```diff
@@ -318,6 +318,134 @@ Function Get-ParamSqlServerRoles
 	return $newparams
 }
 
+
+Function Get-ParamSqlProcessHosts
+{
+<#
+ .SYNOPSIS
+ Internal function. Returns System.Management.Automation.RuntimeDefinedParameterDictionary
```

Comment:
Test comment to show @ctrlbold how line by line commenting works. And to get her to add more comments at the top. :8ball: